src/Collection/Resources/assets/src/controller.ts

```diff
@@ -0,0 +1,98 @@
+import { Controller } from '@hotwired/stimulus';
+
+const DEFAULT_ITEMS_SELECTOR = ':scope > :is(div, fieldset)';
+
+interface CollectionDataset extends DOMStringMap {
+    prototype: string;
+    currentIndex: string;
+    itemsSelector?: string;
+    addButtonTemplateId?: string;
+    disableAddButton?: string;
+    deleteButtonTemplateId?: string;
+    disableDeleteButton?: string;
+}
+
+enum ButtonType {
```

Comment:
This should be a `const enum` instead, to inline the `add` and `remove` string in the emitted JS code instead of emitting an enum object.

Reply:
I changed the code and using `const enum` isn't possible anymore (IIRC, `const enum` causes various issues anyway).